tools/test/audio/std_utils/fr_test_measure.m

```
@@ -33,15 +33,17 @@
 % Author: Seppo Ingalsuo <seppo.ingalsuo@linux.intel.com>
 
 %% Check if upper and lower mask is defined
-if length(test.fr_mask_flo) || length(test.fr_mask_fhi)
+if ~isempty(test.fr_mask_flo) || ~isempty(test.fr_mask_fhi)
 	test.fr_lo = 0;
 	test.fr_hi = 0;
 end
 if test.fr_lo > 0 || test.fr_hi > 0
-	test.fr_mask_flo = [];
```

This to separate commit, e.g. use default frequency response mask.